### PR TITLE
Revive doctest_tb_sysexit

### PR DESCRIPTION
**Pull request by Kojoley**
Output changed with `pure-eval>=0.2.0` which learned to parse binary operators and to show its arguments. Test changed to not use the binary operator '%' to pass on any version.

### CODE REVIEW

**Comment by Kojoley** [comment]
I could not please darker here without touching a lot of unrelated tests.

**Comment by Carreau** [comment]
> I could not please darker here without touching a lot of unrelated tests.

That is weird, I'll try. It may depend on the version of black, and if black changes its formatting.

**Comment by Kojoley** [comment]
The thing is that adding a newline, as darker wants, changes line numbers which is imprinted in other tests.

**Comment by Carreau** [comment]
Ah, I see what you mean. 

I've pushed a fix the the line number, I think it's ok to do that, I don't believe we are actually need to test that the line numbers are the right one.

And just after fixing them I realized that we could actually replace all lines numbers with `...`  that will match anything.

File: IPython/core/tests/test_iplib.py [hunk 1]
```diff
@@ -118,91 +117,100 @@ def doctest_tb_verbose():
...
+    <BLANKLINE>
+    ...sysexit(stat, mode)
+         10 def sysexit(stat, mode):
+    ---> 11     raise SystemExit(stat, f"Mode = {mode}")
+    <BLANKLINE>
+    SystemExit: (2, 'Mode = exit')
+    """
+
+
+def doctest_tb_sysexit_verbose():
```
Comment (Kojoley) [inline IPython/core/tests/test_iplib.py:175]: Is there anywhere else a test that ensures that you can call %tb multiple times with different verbosity?

Comment (Carreau) [reply IPython/core/tests/test_iplib.py:175]: Yes, the one just before switch between plain and context. I separated this one just to make it a tiny bit easier to find the rights lines when fixing doctests.

**Pull request merged**